elastic/kibana
# [Stack Monitoring] De-angularize CCR pages

## Summary

Resolves https://github.com/elastic/kibana/issues/113808

- CCR follower indices list view
- CCR Shard view



### Test
- Connect your local kibana cluster to my cluster that has CCR setup (ping me for credentials).  This only works until the session expires and I think you will need to connect to it before I add the remote cluster so whoever tests this will need to ping me first so I can setup an environment and then they will need to test it immediately.  Alternatively you can  create multiple clusters in your own cloud environment and setup CCR.
- Navigate to Elasticsearch -> Overview -> CCR tab
- You should see 6 follower indices with pagination of 2 pages. (Pagination in the angular version is broken, you might see 6 pages, not fixing it)
- Clicking on a follower index will open a list of shards, there is only one shard per each index, starting with an id of 0.  
- Clicking on the shard id you should take you to the Shard page.  This page has no tab navigation for whatever reason.
- The shard view should have 2 shards and a collpased advanced section
- Expanding the advanced section should have stats that refresh as the page data refreshes with the latest stat
